main.go.tmpl

```diff
@@ -11,6 +11,7 @@
 {{- set $opts "fixEmptyArrays" (ternary (in .Opts.fixEmptyArrays "" "true") true false) -}}
 {{- set $opts "errorStackTrace" (ternary (in .Opts.errorStackTrace "" "true") true false) -}}
 {{- set $opts "legacyErrors" (ternary (in .Opts.legacyErrors "" "true") true false) -}}
+{{- set $opts "webrpcHeader" (ternary (in .Opts.webrpcHeader "" "true") true false) -}}
```

Comment:
I was playing with it, and i think it should look like this? 
```suggestion
{{- set $opts "webrpcHeader" (ternary (in .Opts.webrpcHeader "" "false") false true) -}}
```

Reply:
We agreed to make `webrpcHeader` opt-out instead of `opt-in`. 

This does not generate headers

```
//go:generate webrpc-gen -schema=example.ridl -target=../../../gen-golang -pkg=main -server -client -legacyErrors -fixEmptyArrays -webrpcHeader=false -out=./example.gen.go
```

This does: 
1.
```
//go:generate webrpc-gen -schema=example.ridl -target=../../../gen-golang -pkg=main -server -client -legacyErrors -fixEmptyArrays -webrpcHeader -out=./example.gen.go
``
2. 
```
//go:generate webrpc-gen -schema=example.ridl -target=../../../gen-golang -pkg=main -server -client -legacyErrors -fixEmptyArrays -webrpcHeader=true -out=./example.gen.go
```
3.
```
//go:generate webrpc-gen -schema=example.ridl -target=../../../gen-golang -pkg=main -server -client -legacyErrors -fixEmptyArrays -out=./example.gen.go
```